CHANGELOG.next.asciidoc

```diff
@@ -108,6 +108,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Fix Azure Monitor 429 error by causing metricbeat to retry the request again. {pull}38294[38294]
 - Fix fields not being parsed correctly in postgresql/database {issue}25301[25301] {pull}37720[37720]
 - rabbitmq/queue - Change the mapping type of `rabbitmq.queue.consumers.utilisation.pct` to `scaled_float` from `long` because the values fall within the range of `[0.0, 1.0]`. Previously, conversion to integer resulted in reporting either `0` or `1`.
+- Fix for MySQL/Performance - Query failure in case of MySQL version 5.7 for `quantile_95`. {pull}38710[38710]
```

Comment:
Just 5.7 or all versions prior to 8.0.0 ?

Reply:
All versions prior to 8.0 are affected. We should update this changelog update.

Reply:
To be very precise it was introduced in 8.0.1

See: https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-1.html#:~:text=In%20addition%2C%20the%20events_statements_summary_by_digest%20table%20now%20has%20columns%20that%20provide%20percentile%20information%20about%20statement%20latencies.%20For%20more%20information%2C%20see%20Statement%20Summary%20Tables.

Even Kush pointed it out here: https://github.com/elastic/beats/issues/25069#issuecomment-2029319307 (8.0.1)

Reply:
I will update the Change log, Thanks for bringing this up.